README.md

```diff
@@ -3,7 +3,13 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 ## Getting Started
 
 
-First, run the development server:
+First, install dependencies:
+
+```bash
+npm ci
+```
```

Comment:
Let's try to use yarn here. Sanctuary uses yarn over npm on most of the project :)

```suggestion
First, install dependencies:

```bash
yarn install
```
```

Reply:
Thank you for the feedback @limsohee1002. I'll update and re-submit the PR when I have a chance!